.coveragerc

```diff
@@ -12,10 +12,4 @@ show_missing = true
 
 [run]
 branch = true
-cover_pylib = false
 plugins = Cython.Coverage
-relative_files = true
-source =
-  tests
-source_pkgs =
-  yarl
```

Comment:
Keep this as is, it's configured intentionally. Are there any settings you didn't like?

Reply:
Blindly copying from a repo which still works. I'll just merge the typing fixes and leave it to you.